subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -67,235 +67,134 @@
                 <main class="main-content">
                     <nav class="docs-navigation">
                         <ul>
-                            <li><a href="/userguide/userguide.html">User Manual Home</a></li>
-                            <li><a href="/release-notes.html">Release Notes</a></li>
+                            <li>
+                                <a href="/userguide/userguide.html">User manual</a>
+                                <span style="font-size: 70%;">(
+                                    <a style="display: inline; font-size: 100%;" href="/userguide/userguide_single.html">single page</a>
+                                    |
+                                    <a style="display: inline; font-size: 100%;" href="/userguide/userguide.pdf">PDF</a>)
+                                </span>
+                            </li>
+                            <li><a href="/release-notes.html">Release notes</a></li>
+                            <li><a href="/userguide/installation.html">Installing Gradle</a></li>
                         </ul>
 
-                        <h3 id="getting-started">Getting Started</h3>
+                        <h3 id="tutorials"><a href="https://guides.gradle.org/">Tutorials</a></h3>
+                        <h3 id="reference">Reference</h3>
                         <ul>
-                            <li><a href="/userguide/installation.html">Installing Gradle</a></li>
-                            <li><a href="https://guides.gradle.org/creating-new-gradle-builds/">Creating New Gradle Builds</a></li>
-                            <li><a href="https://guides.gradle.org/creating-build-scans/">Creating Build Scans</a></li>
-                            <li><a href="https://guides.gradle.org/migrating-from-maven/">Migrating From Maven</a></li>
+                            <li><a href="/dsl/">Groovy DSL reference</a></li>
+                            <li><a href="/javadoc/">Gradle API Javadoc</a></li>
+                            <li><a href="/userguide/plugin_reference.html">Plugins</a></li>
```

Comment:
❓ should we make it clearer that this is just for core (bundled) plugins?

Reply:
Probably.

 - [ ] Make it clear that this is a link to the core or built-in plugins